cli/tests/services.bats

```diff
@@ -235,7 +235,7 @@ EOF
 EOF
 )
   assert_success
-  assert_output --partial "✅ Service 'one' restarted"
+  assert_output --partial "✅ Service 'one' started"
```

Comment:
3 of of these modified tests failed in CI: https://github.com/flox/flox/actions/runs/10371132241/job/28711315221?pr=1932#step:6:377

```
# -- output does not contain substring --
# substring (1 lines):
#   ✅ Service 'one' started
# output (2 lines):
#   ✅ Service 'one' restarted.
#   Shutting down process-compose
# --
```

I suspect because the restart began whilst `process-compose` still considered the process as running, even if it finished very shortly afterwards. To guard against it we'd have to poll for the services to reach their final status before issuing the restart.

Reply:
I've been putting a small `sleep 0.1` at the beginning/end of the bash script executed during the activation since your activation is racing `process-compose`. For a real service this probably isn't sufficient, but for our `sleep` and `counter` services it's probably fast enough.

Reply:
That seemed to work in https://github.com/flox/flox/pull/1932/commits/45d9fece378d348a47b33e9939efdb222191a267, so feel free to squash that and merge

Reply:
Thanks for looking! I've gone with a helper script to wait for service statuses because I'm cautious about how easy these tests will be to debug in the future.